subsys/bluetooth/controller/ll_sw/lll_conn_iso.h

```diff
@@ -47,6 +47,9 @@ struct lll_conn_iso_stream {
 	/* Resumption information */
 	uint8_t next_subevent;      /* Next subevent to schedule */
 
+	uint8_t bn_tx;
+	uint8_t bn_rx;
+
```

Comment:
These are already contained in the `rx` and `tx` members - why do we need a copy?

Reply:
these keep track of the current burst number

Reply:
When FT > 1, an SDU would be re-transmitted across ISO intervals until the timeout. `bn_tx` and `bn_rx` in the LLL tracks the current PDU of the SDU to be re-transmitted. `lll_conn_iso_stream_rxtx -> bn`, isnt that the configured BN? 

We move bn_tx/bn_rx as `bn_curr:4` inside `struct lll_conn_iso_stream_rxtx` ?

Reply:
Agreed

Reply:
This is something we (Oticon) handle in LLL without static variables, but no problem, we can store this in the common LLL.